packages/owlbot-bootstrapper/test/test.ts.hbs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
```

Comment:
Another handlebars file

Reply:
I was hoping to keep this one as is because I will end up using this file as a starting point when I fully wire up the bot, but I can also delete it and re-add it later.